samples/file-encryptor/enclave/keys.cpp

```diff
@@ -226,7 +226,12 @@ int ecall_dispatcher::cipher_encryption_key(
     mbedtls_aes_init(&aescontext);
 
     // set aes key
-    ret = mbedtls_aes_setkey_enc(&aescontext, encrypt_key, ENCRYPTION_KEY_SIZE);
+    if (encrypt){
```

Comment:
style: { should be on new line, here and in 232, for consistency with rest of file

Reply:
Done!